json_logic/__init__.py

```diff
@@ -10,7 +10,7 @@ def jsonLogic(tests, data=None):
 
   data = data or {}
 
-  op = tests.keys()[0]
+  op = list(tests.keys())[0]
```

Comment:
list(tests)[0] would be a preferable way to accomplish this. http://blog.labix.org/2008/06/27/watch-out-for-listdictkeys-in-python-3